templates/config.php.j2

```diff
@@ -1,3 +1,25 @@
+{% macro insert_var(var) -%}{#- Formats a variable for inserting it into PHP -#}
+	{%- if var | type == "bool" -%}
+		{#- bool -#}
+		{{ var | bool | lower }}
+	{%- elif var | type == "list" -%}
+		{#- list -#}
+		[ {% for item in var %}{{ insert_var(item) }}, {# Note: trailing commas in lists are legal #}{% endfor %} ]
+	{%- elif var | type == "dict" -%}
+		{#- dict -#}
+		[ {% for k,v in var.items() %}'{{ k }}' => {{ insert_var(v) }}, {# Note: trailing commas in dicts are legal #}{% endfor %} ]
+	{%- elif var | type == "unicode" or var | type == "AnsibleUnsafeText" or var | type == "str" -%}
+		{%- if (var | string).startswith(':nq:') -%}
```

Comment:
I don't like it that the config can, with `:nq:`, run arbitrary code. Why is `:nq:` even needed? Every value that has to be set can be precalculated during or before deployment, so:

* It shouldn't be necessary to calculate config values at run time.
* It's weird design to calculate config values at run time, because then that value could possibly change over time.

Reply:
I left this in because without it, not all values that are used in [the default engelsystem config](https://github.com/engelsystem/engelsystem/blob/master/config/config.default.php) can be specified, e.g. using environment variables as values

Reply:
This role is also responsible for starting the engelsystem. Therefore it controls the environment and there should be no need to access the environment during runtime.